sdcm/logcollector.py

```diff
@@ -806,7 +806,9 @@ class ScyllaLogCollector(LogCollector):
                     CommandLog(name='systemctl.status',
                                command='sudo systemctl status --all --full --no-pager'),
                     CommandLog(name='cassandra-rackdc.properties',
-                               command=f'cat {SCYLLA_PROPERTIES_PATH}')
+                               command=f'cat {SCYLLA_PROPERTIES_PATH}'),
+                    CommandLog(name='scripts_debug.log',
+                               command='for i in /var/tmp/scylla/*.log;do echo [$i]; cat $i; sudo rm $i;done')
```

Comment:
Please don't delete the logs.

We never do that, there's no point since that machine is going to be taken down
And if we manually want to connect and investigate, those would be missing.

Reply:
Umm, current implementation of script debug log is just generate log to "/var/tmp/scylla/${SCRIPT_NAME}-${PID}-debug.log", if we run test multiple time we cannot determine which log is generated on current test.

Maybe we need add an feature to specify log directory by configuration file, and change the directory from /var/tmp/scylla to current SCT working directory.

Reply:
SCT doesn't run the test multiple time on the same VM, it always recreate new VM.
same goes to all of the logs.

the only reason I'm asking is for case we manual connect, we should be removing any data or information from node under test.

in the end we collect logs per node, not per install/setup step we are doing. and that's o.k.

Reply:
Stopped dropping logs.